xmodule/modulestore/tests/django_utils.py

```diff
@@ -613,6 +613,35 @@ def update_course(self, course, user_id):
         return updated_course
 
 
+class ImmediateOnCommitMixin:
+    """
+    Mixin for tests that want `on_commit` callbacks to run immediately,
+    even under TestCase (which normally wraps tests in a transaction
+    that never commits).
+    Especially useful when the test needs to execute an event that occurs after an `on_commit`
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        super_cls = super(ImmediateOnCommitMixin, cls)
```

Comment:
```suggestion
        super_cls = super()
```

We don't need to specify class name here?

Reply:
Updated https://github.com/openedx/edx-platform/pull/37485/commits/f7cd6cdac622d166048e653ae97ca9e2ca812bc2